### PR TITLE
feat: show new events and fetch correctly

### PR DESCRIPTION
**Pull request by NicoSantangelo**
It's not trying to be 100% complete with the descriptions

### CODE REVIEW

File: scripts/mktcli.js [hunk 1]
```diff
@@ -223,13 +223,13 @@ const main = {
         asSafeAction(async (assetId, assetType, options) => {
           const asset = await getAssetFromCLIArgs(assetId, assetType)
 
-          const blockchainEvents = await BlockchainEvent.findByArgs(
-            'assetId',
+          const blockchainEvents = await BlockchainEvent.findByAnyArgs(
+            ['assetId', 'landId', '_landId', '_estateId'],
```
Comment (nachomazzara) [inline scripts/mktcli.js]: Also we have `_tokenId` for the ERC721 Transfer event

File: scripts/mktcli.js [hunk 2]
```diff
@@ -290,7 +304,7 @@ const main = {
           const asset = await getAssetFromCLIArgs(assetId, assetType)
 
           const events = await BlockchainEvent.findByAnyArgs(
-            ['assetId', '_landId'],
+            ['assetId', 'landId', '_landId', '_estateId'],
```
Comment (nachomazzara) [inline scripts/mktcli.js]: Maybe we could add also `_tokenId`

Comment (nachomazzara) [reply scripts/mktcli.js]: Hmm What do you think if we create a const like

`const argsId = ['assetId', 'landId', '_landId', '_estateId', '_tokenId']`

IDK a good name: `argsId`, `wordToMatchId`, `matchId`, `argsNames`

Comment (NicoSantangelo) [reply scripts/mktcli.js]: Agree with abstracting this!

**Comment by NicoSantangelo** [comment]
Updated! let me know what you think of the constant name! (POSSIBLE_ASSET_EVENT_IDS)

**Comment by nachomazzara** [comment]
giphy

**Pull request merged**